tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -227,7 +227,7 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                       jobs["ec2"] = { build job: 'rancher-v3_ontag_ec2_certification', parameters: params }
                       jobs["az"] = { build job: 'rancher-v3_ontag_az_certification', parameters: params }
                       jobs["custom"] = { build job: 'rancher-v3_ontag_custom_certification', parameters: params}
-                      jobs["go-provisioning"] = { build job: 'go-automation-freeform-job', parameters: goParams }
+                      jobs["go-provisioning"] = { build job: 'rancher-v3_ontag_go_certification', parameters: goParams }
```

Comment:
since the go tests aren't v3, can you remove that from the name?

Reply:
done